lib/repl.js

```diff
@@ -336,7 +336,7 @@ function REPLServer(prompt,
           filename: file,
           displayErrors: true,
           importModuleDynamically: async (specifier) => {
-            return asyncESM.ESMLoader.import(specifier, pwd);
+            return asyncESM.ESMLoader.import(specifier, pwd + '/repl');
```

Comment:
```suggestion
            return asyncESM.ESMLoader.import(specifier, path.join(pwd, 'repl'));
```

Reply:
If I use the `path` module here do I need to use `path.posix.join` since this is a URL?  Maybe better to `pwd = pathToFileURL(path.join(process.cwd(), 'repl')).href;` in the first place?  Additionally this is the only thing `pwd` is used for so I could rename it to `referrer` or `parent` to avoid confusion?

Reply:
I've done the path.join before conversion to URL and renamed the variable to `parentURL`.  Hope this is OK, if not I'll modify as requested.